pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
     "jsonschema>=4.6,<5.0",
     "libcomps>=0.1.20.post1,<0.2",
     "productmd~=1.33.0",
-    "pulpcore>=3.70.0,<3.85",
+    "pulpcore>=3.71,<3.85",
```

Comment:
I think this is not exactly what we want. Should we add pulpcore 3.70 to the supported branches instead?

Reply:
This is the "don't use unsupported branches as LBs" change. Nobody "requires" 3.70 - so this updates to current-latest, unless/until we call something out as "supported".